addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -433,7 +433,7 @@ export class WebsitePreview extends Component {
     _cleanIframeFallback() {
         // Remove autoplay in all media video iframes urls so videos are not
```

Comment:
Maybe adapt the comment to just mention "all iframes" ?

Reply:
:+1: done